airflow/models/dagbag.py

```diff
@@ -406,8 +404,10 @@ def collect_dags(
         FileLoadStat = namedtuple(
             'FileLoadStat', "file duration dag_num task_num dags")
 
+        from airflow.utils.file import correct_maybe_zipped
         dag_folder = correct_maybe_zipped(dag_folder)
 
+        from airflow.utils.file import list_py_file_paths
```

Comment:
Small nit: we could combine the two imports in to a single line.

Reply:
Fixed those as well.